sfn.go

```diff
@@ -26,8 +28,8 @@ type StreamFunction interface {
 	Connect() error
 	// Close will close the connection
 	Close() error
-	// Init will initialize the stream function
-	Init(fn func() error) error
+	// Wait waits sfn to finish.
+	Wait()
```

Comment:
Is it better to return an `error`?

Reply:
The `error` has been send to `ErrHandler`, `Wait` just wait sfn escape the for-loop.